datahub-web-react/src/app/homeV3/module/components/LargeModuleFallback.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { FallbackProps } from 'react-error-boundary';
+
+import EmptyContent from '@app/homeV3/module/components/EmptyContent';
+import LargeModule from '@app/homeV3/module/components/LargeModule';
+import { ModuleProps } from '@app/homeV3/module/types';
+
+interface Props {
+    fallbackProps: FallbackProps;
+    moduleProps: ModuleProps;
+}
+
+export default function LargeModuleFallback({ moduleProps, fallbackProps }: Props) {
+    return (
+        <LargeModule {...moduleProps}>
+            <EmptyContent
+                title="Oops!"
+                description="Something didn't go as planned. Our team has been notified of this error and is working on a resolution. Try refreshing."
```

Comment:
i'm a little nervous about the copy saying our team has been notified and we're working on a resolution since i don't think that's actually the case.

maybe instead we say "Something didn't go according to plan with this module. Try refreshing or contacting your DataHub Administrator"